src/data/matrix/basic.lean

```diff
@@ -1370,31 +1370,41 @@ lemma transpose_map {f : α → β} {M : matrix m n α} : Mᵀ.map f = (M.map f)
 by { ext, refl }
 
 /-- `matrix.transpose` as an `add_equiv` -/
+variables (m n α)
```

Comment:
```suggestion
variables (m n α)
/-- `matrix.transpose` as an `add_equiv` -/
```
otherwise I'm not sure if the comment reaches its intended target.

Reply:
Sorted.